_clients/opensearch-py-ml.md

```diff
@@ -0,0 +1,35 @@
+---
+layout: default
+title: Opensearch-py-ml
+nav_order: 11
+---
+
+`opensearch-py-ml` is a Python client that provides a suite of data analytics and machine learning (ML) tools for OpenSearch. It provides data analysts with the following abilities:
+
+- Call OpenSearch indexes and manipulate each index as a [pandas.DataFrame](https://pandas.pydata.org/docs/reference/api/pandas.DataFrame.html). These data frames support complex filtering and aggregation operations.
+- Provides basic support to upload models into OpenSearch using the [ML Commons plugin]({{site.url}}{{site.baseurl}}/ml-commons-plugin/index/)
+
+## Install `opensearch-py-ml`
+
+To add the client to your project install it using [pip](https://pip.pypa.io/):
+
+```bash
+pip install opensearch-py-ml
+```
+
+Then, import the client into OpenSearch like any other module:
+
+```python
+import openseach_py_ml as oml
+from opensearch-py-ml import OpenSearch
```

Comment:
We should mention that, we have a dependency on `opensearchpy` plugin to connect with opensearch.

Then we can do:

```
from opensearchpy import OpenSearch
import openseach_py_ml as oml
```

Reply:
I think `opensearchpy` should be opensearch python client, not plugin

Reply:
Yeah agree.